src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,26 @@
+package duke.task;
+
+/**
+ * Represents a Task object. It is an abstract class.
+ */
+public abstract class Task {
+    public abstract String showTask();
+    public abstract String showTaskOnly();
```

Comment:
Maybe consider renaming this method to reflect that you're showing the task name only as the implementations seem to reflect this?